cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=60, required=True)
```

Comment:
The `title` field in the serializer has a `max_length` of 60, while in the `Movie` model, it is set to 100. Ensure that the `max_length` in the serializer matches the model to avoid validation issues.